plugins/browser-plugin-media-tracking/src/helperFunctions.ts

```diff
@@ -53,7 +53,7 @@ type TimeRange = { start: number; end: number };
 export function timeRangesToObjectArray(t: TimeRanges): TimeRange[] {
   const out: TimeRange[] = [];
   for (let i = 0; i < t.length; i++) {
-    out.push({ start: t.start(i), end: t.end(i) });
+    out.push({ start: t.start(i) || 0, end: t.end(i) || 0 });
```

Comment:
Should we also handle the [`Infinity` case](https://html.spec.whatwg.org/multipage/media.html#seeking:dom-media-duration) for `end`, which would make it `null` and also invalid?

> whose `end` is the same as the time of the first frame plus the `duration` attribute's value (which would equal the time of the last frame, and might be positive `Infinity`)